build.sbt

```diff
@@ -30,12 +31,19 @@ inThisBuild(Def.settings(
   ),
   testOptions += Tests.Argument(TestFrameworks.JUnit, "-q", "-v"),
   Dependencies.Versions,
-  Formatting.formatSettings
+  Formatting.formatSettings,
+  shellPrompt := { s => Project.extract(s).currentProject.id + " > " }
 ))
 
+val dontPublishSettings = Seq(
```

Comment:
Heh, I see though the plugin is wrong it seems - needs to cancel publishSigned - the one we actually care about. Will change the PR